e2e/nomostest/nt.go

```diff
@@ -966,3 +968,39 @@ func cloneCloudSourceRepo(nt *NT, repo string) (string, error) {
 	}
 	return cloneDir, nil
 }
+
+// ListReconcilerRoleBindings is a convenience method for listing all RoleBindings
+// associated with a reconciler.
+func (nt *NT) ListReconcilerRoleBindings(syncKind string, rsRef types.NamespacedName) ([]rbacv1.RoleBinding, error) {
+	opts := &client.ListOptions{}
+	opts.LabelSelector = client.MatchingLabelsSelector{
```

Comment:
slightly redundant to do this twice. You could have a function that returns the label selector. in fact, I'm surprised there isn't one under `pkg/reconcilermanager/controllers` already. Don't we need it there too for cleanup?

Reply:
There is one but it's a method of `reconcilerBase`. I suppose I can refactor that to a function